options/locale/locale_en-US.ini

```
@@ -776,6 +780,11 @@ migrate_options = Migration Options
 migrate_service = Migration Service
 migrate_options_mirror_helper = This repository will be a <span class="text blue">mirror</span>
 migrate_options_mirror_disabled = Your site administrator has disabled new mirrors.
+migrate_options_lfs = Migrate LFS files
+migrate_options_lfs_endpoint.label = LFS Endpoint
+migrate_options_lfs_endpoint.description = Migration will attempt to use your Git remote to determine the LFS server. You can also configure a custom LFS server.
```

Minor nit. I read this as if it were configuring (initializing) an LFS server at the endpoint in form input.

```suggestion
migrate_options_lfs_endpoint.description = Migration will attempt to use your Git remote to determine the LFS server. You can also point to a custom LFS server.
```

---

The wording was taken from [here](https://github.com/git-lfs/git-lfs/blob/main/docs/api/server-discovery.md#server-discovery).
How about that?
`You can also configure a custom LFS endpoint if the repository data is stored somewhere else.`